tests/repository/user/test_user.py

```diff
@@ -151,3 +152,30 @@ async def test_find_by_invalid_oauth_id(self, session: AsyncSession, user_fixtur
 
         # then
         assert not result
+
+    @pytest.mark.asyncio
+    async def test_find_all_by_nickname(self, session: AsyncSession, user_fixture: User, other_user_fixture: User):
+        # given
+        nickname = "name"
+        other_name = "wrong"
+        params = Params(page=1, size=10)
+
+        # then
+        assert await user_repository.find_all_by_nickname(
+            session, nickname=nickname, params=params
+        ) == Page.create(items=[user_fixture, other_user_fixture], params=params, total=2)
+        assert await user_repository.find_all_by_nickname(
+            session, nickname=other_name, params=params
+        ) == Page.create(items=[], params=params, total=0)
+
+    @pytest.mark.asyncio
+    async def test_find_by_ids(self, session: AsyncSession, user_fixture: User, other_user_fixture: User):
+        # given
+        ids = [user_fixture.id, other_user_fixture.id]
+
+        # when
+        result = await user_repository.find_by_ids(session, ids=ids)
+        result.sort(key=lambda x: x.nickname)
```

Comment:
result에 꽂히는 User의 순서가 랜덤이기 때문에, 일단 nickname으로 sorting해서 테스트 넘어가게는 해놨는데,, 이거 하나 통과하자고 쿼리문에 orderby를 추가하는게 맞나 싶어서.. 일단 나온 User 리스트에 대해서 따로 sorting을 했는데, 혹시 더 좋은 방법이 있을까요?

Reply:
이런 식으로 하는 것이 좋을 것 같아요
```suggestion
        # when
        result = await user_repository.find_by_ids(session, ids=ids)

        # then
        assert set(result) == {user_fixture, other_user_fixture}
```